scripts/docker/buildpack-deps/Dockerfile.emscripten

```
@@ -51,21 +54,21 @@ RUN set -ex; \
 		-DZ3_BUILD_EXECUTABLE=OFF \
 		-DZ3_SINGLE_THREADED=ON \
 		-DCMAKE_CXX_FLAGS="-s DISABLE_EXCEPTION_CATCHING=0" \
-		..; \
-	make ; make install; \
-	rm -r /usr/src/z3; \
-	cd /usr/src; \
-
-	wget -q 'https://boostorg.jfrog.io/artifactory/main/release/1.75.0/source/boost_1_75_0.tar.bz2' -O boost.tar.bz2; \
-	test "$(sha256sum boost.tar.bz2)" = "953db31e016db7bb207f11432bef7df100516eeb746843fa0486a222e3fd49cb  boost.tar.bz2"; \
-	tar -xf boost.tar.bz2; \
-	rm boost.tar.bz2; \
-	cd boost_1_75_0; \
-	mv ../emscripten.jam .; \
-	./bootstrap.sh; \
-	echo "using emscripten : : em++ ;" >> project-config.jam ; \
+		.. && \
+	make && make install && \
```

```suggestion
	make && \
	make install && \
```
Actually `make install` alone would probably be enough but if this works, let's keep it.